test/plugins/setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+setup(
+    name='plugins',
+    version='1.0',
+    author='Dalibo',
+    author_email='contact@dalibo.com',
+    license='PostgreSQL',
+    install_requires=['temboard-agent'],
+    py_modules=['plugins'],
+    entry_points={
+        'temboardagent.plugins': [
+            'hellong = plugins:Hello',
+        ],
+    },
+)
```

Comment:
Why this file ? Does that mean we'll need to refer each plugin as an entry_point in this file ?

Reply:
test/plugins est un projet bidon de greffons externes. Pour ne pas polluer le code interne avec pour le moment, quand je pourrais remplacer le code de hello.